src/ks-clock-face.c

```diff
@@ -1,13 +1,20 @@
 #include <pebble.h>
 
-#define COLORS       PBL_IF_COLOR_ELSE(true, false)
+#define COLORS PBL_IF_COLOR_ELSE(true, false)
 #define ANTIALIASING true
 
-#define HAND_MARGIN  10
-#define FINAL_RADIUS 55
+#define HAND_MARGIN 10
+
+#if PBL_DISPLAY_WIDTH == 200
```

Comment:
Is there any way to calculate what you need using `layer_get_unobstructed_bounds` and `grect_inset`?